app/assets/javascripts/client/inventories/analysis_dashboard.html

```diff
@@ -0,0 +1,70 @@
+<div class="dashboard">
+  <div class="row row-bg row-header">
+    <div class="col-md-12">
+      <h1 class="greeting">{{analysisDashboard.inventory.name}}</h1>
+      <h4 class="byline">organized by: <a href="#">{{analysisDashboard.inventory.facilitator.full_name}} </a>
+        on {{analysisDashboard.inventory.created_at | amDateFormat:'MMMM Do, YYYY'}}
+      </h4>
+    </div>
+  </div>
+  <div class="row">
+    <div id="participants" ng-if="analysisDashboard.currentAnalysis.is_facilitator">
+      <div class="row">
+        <div class="col-md-12 header">
+          <h2 class="participant-count"
+              ng-pluralize count="analysisDashboard.currentAnalysis.participant_count"
+                           when="{'0': 'No Participants',
+                                  '1': '{} Participant',
+                                  'other': '{} Participants'}">
+          </h2>
+        </div>
+      </div>
+      <hr class="dashboard-hr" />
+      <div class="col-md-6 col-sm-6" ng-repeat="user in analysisDashboard.participants">
+        <participant user="user"></participant>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-lg-12">
+        <add-inventory-users-link send-invite="true"></add-inventory-users-link>
+      </div>
+      <div class="col-lg-12">
+        <manage-inventory-permissions-link auto-show="false"
+                                           send-invite="true"
+                                           number-of-requests="{{analysisDashboard.inventory.number_of_requests}}">
+        </manage-inventory-permissions-link>
+      </div>
+      <div class="col-lg-12">
+        <invite-inventory-user-link send-invite="true"
+                                    role="participant">
+        </invite-inventory-user-link>
+      </div>
+    </div>
+    <div id="messages" ng-if="analysisDashboard.inventory.messages">
+      <div class="col-md-12 header">
+        <h2 ng-pluralize count="analysisDashboard.inventory.messages.length"
+                          when="{'0': 'No Sent Messages',
+                                 '1': '{} Sent Message',
+                                 'other': '{} Sent Messages'}">
+
+        </h2>
+      </div>
+      <div class="col-md-12" ng-repeat="message in analysisDashboard.inventory.messages">
```

Comment:
This shouldn't be pulling the inventory's messages.  An analysis has its own messages that it can pull in.